src/components/Item/ListItem/Item.js

```diff
@@ -11,8 +12,6 @@ import { isEditMode } from '../../../modules/dashboardModes'
 import { itemTypeMap, getItemUrl } from '../../../modules/itemTypes'
 import { orArray } from '../../../modules/util'
 import ItemHeader from '../ItemHeader/ItemHeader'
-import DeleteIcon from './assets/Delete'
-import DescriptionIcon from './assets/Description'
```

Comment:
Is this icon used somewhere else? I don't see it removed from the repo in this PR

Reply:
Right. Removed.